controllers/comments.controllers.js

```diff
@@ -0,0 +1,13 @@
+const { request } = require("express");
```

Comment:
You don't need express required into your controller. You are already requiring this into the app which is where these functions are being exported too 🙂